lib/core/src/server/manager/manager-config.js

```diff
@@ -36,6 +36,9 @@ const getAutoRefs = async (options) => {
   return list.filter(Boolean);
 };
 
+// strip trailing slash from composition URL
+const toUrl = (url) => url.replace(/\/$/, '');
```

Comment:
`toUrl` ... is this converting a URL .. to a URL?

Reply:
I'll change it to `stripTrailingSlash`?